crates/bvs-registry/src/contract.rs

```diff
@@ -401,6 +449,24 @@ mod query {
         Ok(status.into())
     }
 
+    /// Get the registration status of an operator to a service at a specific block height
+    /// returns: [`StdResult<StatusResponse>`]
+    ///
+    /// #### Warning
+    /// This function will return old data
+    /// if height is equal to the height of the save operation.
+    /// New data will only be available at height + 1
```

Comment:
```suggestion
    /// This function will return previous state
    /// if height is equal to the height of the save operation.
    /// The new state will only be available at height + 1
```

Should use previous, instead of old to describe transition.

I would also describe this "end of block" update, so the re-ordering of the transaction cannot game the system.